index.js

```diff
@@ -8,6 +8,71 @@ const checkWIPModule = require('./lib/checkWipDraftPR');
 
 const whitelistedAccounts = (
   (process.env.WHITELISTED_ACCOUNTS || '').toLowerCase().split(','));
+const checksWhitelist = {
+  'oppia-android': {
+    'opened': ['cla-check'],
+    'reopened': [],
+    'labeled': [],
+    'synchronize': [],
+    'closed': [],
+    'edited': []
+  },
+  'oppia': {
+    'opened': [
+      'cla-check',
+      'changelog-check',
+      'branch-check',
+      'wip-check'],
+    'reopened': [
+      'changelog-check',
+      // Prevent user from reopening the PR.
+      'branch-check',
+      'wip-check'
+    ],
+    'labeled': ['assignee-check',],
+    'synchronize': ['merge-conflict-check'],
+    'closed': ['all-merge-conflict-check'],
+    'edited': ['wip-check']
+  }
+};
+
+async function runChecks(context, checkContext) {

```

Comment:
Instead of ``checkContext`` maybe 'event' would be clearer? It seems like each context here is really an event.

Reply:
Done.